detect_secrets/core/upgrades/v1_2.py

```diff
@@ -0,0 +1,6 @@
+from typing import Any
+from typing import Dict
+
+
+def upgrade(baseline: Dict[str, Any]) -> None:
+    return
```

Comment:
@domanchi @KevinHock Hello. I am planning on doing a release for detect-secrets. From the last version I don't see any changes to the baseline file. Other than 2 new plugins added - But I don't think that needs to be added here. Can you advise?

Reply:
@jpdakran : a noop upgrade file should not be needed. The upgrade logic imports all modules, and intelligently compares their name to the version that it needs to bump to. This means that if no file exists, it will be the same as a noop.

See https://github.com/Yelp/detect-secrets/blob/master/detect_secrets/core/baseline.py#L115-L134 for more details.